.pre-commit-config.yaml

```diff
@@ -0,0 +1,22 @@
+repos:
+- repo: local
```

Comment:
We can leverage the pre-commit configs of other repos to avoid duplicating their configuration.

Reply:
My experience with not using local is that it always screws things up somehow. However, I haven't tried this in a while. I suggest we merge and then try out using remote configs.

WDYT?

Reply:
> My experience with not using local is that it always screws things up somehow.

I haven't bumped into that, do you have more details or an example?

I think we should go with the more conventional documented approach -- which is using the library provided configs -- unless we can demonstrate an issue with them.

Reply:
nm. Updated as per your suggestion.